clickhouse-http-client/src/main/java/com/clickhouse/client/http/ClickHouseHttpClient.java

```diff
@@ -28,6 +30,57 @@ public class ClickHouseHttpClient extends AbstractClient<ClickHouseHttpConnectio
 
     static final List<ClickHouseProtocol> SUPPORTED = Collections.singletonList(ClickHouseProtocol.HTTP);
 
+    public static class HostNameAndAddress{
+        public String hostName;
+        public String address;
+    }
+
+    private static HostNameAndAddress getLocalHost() {
+        // get local address but not localhost
+        HostNameAndAddress hostNameAndAddress = new HostNameAndAddress();
+        try {
+            Enumeration<NetworkInterface> networkInterfaces = NetworkInterface.getNetworkInterfaces();
+
+            for (NetworkInterface ni : Collections.list(networkInterfaces)) {
+                Enumeration<InetAddress> inetAddresses = ni.getInetAddresses();
+                for (InetAddress ia : Collections.list(inetAddresses)) {
+                    // We just use the first non-loopback address
+                    if (!ia.isLoopbackAddress()) {
+                        hostNameAndAddress.address = ia.getHostAddress();
+                        hostNameAndAddress.hostName = ia.getCanonicalHostName();
+                        break;
+                    }
+                }
+            }
+        } catch (SocketException e) {
+            // ignore
+        }
+        return hostNameAndAddress;
+    }
+
+    public static String getReferer(ClickHouseConfig config) {
+        String referer = null;
+        if (!ClickHouseChecker.isNullOrEmpty(config.getStrOption(ClickHouseHttpOption.SEND_HTTP_CLIENT_ID)))
+        {
+            if (ClickHouseChecker.isNullOrEmpty(ClickHouseOption.toKeyValuePairs(config.getStrOption(ClickHouseHttpOption.CUSTOM_HEADERS)).get("referer"))) {
+                if (config.getStrOption(ClickHouseHttpOption.SEND_HTTP_CLIENT_ID).equals("HOST_NAME"))
+                    referer = LOCAL_HOST.hostName;
+                else
+                    referer = LOCAL_HOST.address;
+            }
+        }
+        return referer;
+    }
+
+    public static HostNameAndAddress LOCAL_HOST = null;
```

Comment:
please make it an instance variable

Reply:
`LOCAL_HOST` is needed in both in `ClickHouseHttpClient` and `ClickHouseHttpConnection`, because `ClickHouseHttpConnection.createDefaultHeaders` is invoked both there.

So I made` LOCAL_HOST` as a static field.

Reply:
I think, I've missed something. 
LOCAL_HOST is accessed only in the `ClickHouseHttpClient.getReferer(config)`.  
 `ClickHouseHttpConnection.createDefaultHeaders` uses already baked value. 

Or do you mean it is used in tests also? ClickHouseHttpConnectionTest

Thanks!